app/controllers/admin/tracks_controller.rb

```diff
@@ -59,10 +60,50 @@ def toggle_cfp_inclusion
       end
     end
 
+    def restart
+      update_state(:restart, 'Review started!')
+    end
+
+    def ready_to_accept
+      update_state(:readiness_to_accept, 'Track request is signaling readiness for acceptance!')
```

Comment:
Can't help myself thinking of 
http://top40-charts.com/songs/lyrics.php?sid=28986&string=Catch%20Me%20When%20I%27m%20Falling

Perhaps in our case we are not signaling anything to anyone, but rather marking the track as a possible acceptance :)